README.md

```diff
@@ -210,7 +212,7 @@
 
 ## Audition
 
-### General
+### General2
```

Comment:
No need to add numbers to the heading, they'll be in different subheadings anyway.
```suggestion
### General
```